src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs

```diff
@@ -195,6 +195,7 @@ internal void FormatWorkerPathIfNeeded(ISystemRuntimeInformation systemRuntimeIn
             DefaultWorkerPath = DefaultWorkerPath.Replace(RpcWorkerConstants.OSPlaceholder, os.ToString())
                              .Replace(RpcWorkerConstants.ArchitecturePlaceholder, architecture.ToString())
                              .Replace(RpcWorkerConstants.RuntimeVersionPlaceholder, DefaultRuntimeVersion);
+            logger.LogDebug($"DefaultWorkerPath: {DefaultWorkerPath}");
```

Comment:
This information is already logged here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs#L50